examples/all-clusters-app/esp32/main/CHIPDeviceManager.cpp

```diff
@@ -99,6 +99,13 @@ CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
 void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath & path, uint8_t mask, uint8_t type, uint16_t size,
                                        uint8_t * value)
 {
+    TaskHandle_t task = xTaskGetCurrentTaskHandle();
+    const char * name = pcTaskGetName(task);
+    if (!strcmp(name, "CHIP"))
+    {
+        ESP_LOGE("all-clusters-app", "Attribute changed on non-Matter task '%s'\n", name);
+    }
```

Comment:
Do we need to commit this too?

Reply:
I would like to, so we catch more instances of this if they happen, yes..